test/spec.js

```diff
@@ -1259,6 +1313,61 @@ module.exports = (fetchMock, theGlobal, Request, Response) => {
 			});
 		})
 
+		describe('includeContentLength', () => {
+			it('should work on body of type object', done => {
+				fetchMock.mock('http://it.at.there/', {body: {hello: 'world'}, includeContentLength: true});
+				fetch('http://it.at.there/')
+					.then(res => {
+						expect(res.headers.get('content-length')).to.equal('17');
+						done();
+					});
+			});
+
+			it('should work on body of type string', done => {
+				fetchMock.mock('http://it.at.there/', {body: 'Fetch-Mock rocks', includeContentLength: true});
+				fetch('http://it.at.there/')
+					.then(res => {
+						expect(res.headers.get('content-length')).to.equal('16');
+						done();
+					});
+			});
+
+			it('should not overrule explicit mocked content-length header', done => {
+				fetchMock.mock('http://it.at.there/', {
+					body: {
+						hello: 'world'
+					}, 
+					headers: {
+						'Content-Length': '100',
+					},
+					includeContentLength: true
+				});
+				fetch('http://it.at.there/')
+					.then(res => {
+						expect(res.headers.get('content-length')).to.equal('100');
+						done();
+					});
+			});
+
+			it('should ignore case of explicit content-length header', done => {
```

Comment:
confusing title for the test - I read it as case meaning 'in the case that'. How about `should be case-insensitive when checking for explicit content-length header`